target/linux/ramips/image/mt7621.mk

```diff
@@ -1616,6 +1627,17 @@ define Device/youku_yk-l2
 endef
 TARGET_DEVICES += youku_yk-l2
 
+define Device/yuncore_ax820
+  $(Device/dsa-migration)
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := YunCore
+  DEVICE_MODEL := AX820
+  DEVICE_PACKAGES := kmod-mt7915e
+  IMAGES += factory.bin
+  IMAGE/factory.bin := append-kernel | append-rootfs | pad-rootfs | yuncore-ax820-metadata | check-size
```

Comment:
Max. 74 chars per line.

Reply:
removed due to removed yuncore-ax820-metadata hack.